chaos-monkey-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -99,6 +99,50 @@ CAUTION: Each Chaos Monkey version is built for a specific Spring Boot version,
 
 It is your decision how you want to integrate and use the CM4SB. There is no difference between the two possibilities. However, it's easier to add it as a dependency (see <<_as_dependency>>), so you don't need to adjust the starting of your application.
 
+=== Preparing your application
+
+If you want chaos monkey to attack your classes they need to be open (=non-final). If you're using java, this is the default. Just don't mark your classes or methods as final. If you're using kotlin, we recommend using the kotlin spring plugin:
```

Comment:
I think the language names should be uppercased (Java, Kotlin) Also a short summary describing what the plugin does would help.

> ...we recommend using the Kotlin spring plugin which tells the compiler to make classes with Spring annotations open (`@Controller`, `@Service`, etc.):